src/test/java/com/google/devtools/build/lib/query2/cquery/ProtoOutputFormatterCallbackTest.java

```diff
@@ -29,11 +29,13 @@
 import com.google.devtools.build.lib.events.NullEventHandler;
```

Comment:
Could you please make sure this test target build after the change? I did an import changelist and it did not build successfully. Even though the checks in this PR seem to have passed, the test for `ProtoOutputFormatterCallbackTest` was not included looking at the artifacts (only TransitionsOutputFormatterTest was tested). I'll do a bit more debugging tomorrow morning.

Reply:
It worked for me in open source:
```
//src/test/java/com/google/devtools/build/lib/query2/cquery:ProtoOutputFormatterCallbackTest PASSED in 32.9s
```

I introduced a compile break to it, and it stopped compiling, so in a Bazel world this looks fine... I'm not sure what would be different internally, sorry!